x11-packages/xorg-mkfontscale/postinst

```diff
@@ -7,8 +7,7 @@ if [ -d "${PREFIX}/share/fonts" ]; then
         [ ! -d "${dir}" ] && continue
```

Comment:
```suggestion
        [[ ! -d "${dir}" ]] && continue
```

Reply:
Also the whole file should be reindented to use tabs for consistency with other scripts.